README.md

```diff
@@ -1 +1,65 @@
-# Awesome-Books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+"Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list. By building this application, we learned how to manage data using JavaScript. Thanks to that your website will be more interactive. We used a medium-fidelity wireframe to build the UI.
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link]
+
+
+## Getting Started
+
+### Prerequisites
+* Have git installed. (For version control)
+* Have text editor installed. (VSCode, sublime, atom)
+* Have CSS,HTML & JS Linters installed and configured. For further information, visit [this link](https://github.com/microverseinc/linters-config/blob/master/README.md) to get better understanding on each linters used on this project.
+
+### Usage
+This page registers books with author and title of the book. To register a book:
+1. Type the author's name.
+2. Type the book's title.
+3. Click on Add button below.
+4. The books will be added to the list and will be displayed.
+
+To remove a book from the list:
+1. Find the book in the list.
+2. Click on remove button for the respective book.
+
+## Authors
+
+👤 **Wali Muhamad**
+
+- GitHub: @Itswali
+- Twitter: @Wali-Muhammad
+- LinkedIn: @wali-Muhammad
+
+👤 **Ivan Martinez von Halle**
+
+- GitHub: [@ivanmvh](https://github.com/ivanmvh)
+- Twitter: [@imprivado](https://twitter.com/imprivado)
+- LinkedIn: [Iván Martínez von Halle] (https://www.linkedin.com/in/ivan-martinez-von-halle/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+<!-- ## Acknowledgments
```

Comment:
- [OPTIONAL] I think removing unused comments from your files is better. I suggest creating the `Acknowledgements` section quoting those you want to thank. Your README will look more professional to those who check your work.